hack/local-up-cluster.sh

```diff
@@ -638,7 +638,7 @@ function start_controller_manager {
       --cluster-signing-cert-file="${CLUSTER_SIGNING_CERT_FILE}" \
       --cluster-signing-key-file="${CLUSTER_SIGNING_KEY_FILE}" \
       --enable-hostpath-provisioner="${ENABLE_HOSTPATH_PROVISIONER}" \
-      "${node_cidr_args[@]}" \
+      ${node_cidr_args[@]+"{node_cidr_args[@]}"} \
```

Comment:
Thank you for fix.
I think that this code needs "$" like other changes.

Reply:
Fixed. Thanks!